tasks/main.yml

```diff
@@ -25,11 +25,29 @@
     dest: "/etc/systemd/system/appium.service"
     mode: 0664
 
+- name: "Register appium-webdriveragent folder"
```

Comment:
nit: "Stat appium-webdriveragent folder" or "Get permissions of appium-webdriveragent folder"?